Package.swift

```diff
@@ -46,7 +46,7 @@ let package = Package(
             dependencies: []),
         .testTarget(
             name: "RegexTests",
-            dependencies: ["_StringProcessing"]),
+            dependencies: ["_StringProcessing", "Algorithms"]),
```

Comment:
Related: @timvermeulen any progress on vetting these API for the `_StringProcessing` model?